public/server/db.json

```
@@ -185,10 +186,48 @@
   "fgs": {
     "definition": "For God's sake"
   },
+  
+  "tysm": {
+    "definition": "Thank You So Much"
+  },
+
+  "ty": {
+    "definition": "Thank You"
+  },
+
+  "syl": {
+    "definition": "See You Later"
+  }, 
+
+  "hbty": {
+  	"definition": "Happy Birthday To You"
+  }, 
+
+  "hbu": {
+  	"definition": "How About You"
+  }, 
+
+  "imy": {
+  	"definition": "I Missed You"
```

I think it's `miss` here. 
```suggestion
  	"definition": "I Miss You"
```